Modix/ModixBotHooks.cs

```diff
@@ -45,40 +47,18 @@ public Task HandleLog(LogMessage message)
 
         public async Task HandleAddReaction(Cacheable<IUserMessage, ulong> message, ISocketMessageChannel channel, SocketReaction reaction)
         {
-            await codePaste.ReactionAdded(message, channel, reaction);
+            await _codePaste.ReactionAdded(message, channel, reaction);
         }
 
         public async Task HandleRemoveReaction(Cacheable<IUserMessage, ulong> message, ISocketMessageChannel channel, SocketReaction reaction)
         {
-            await codePaste.ReactionRemoved(message, channel, reaction);
+            await _codePaste.ReactionRemoved(message, channel, reaction);
         }
 
-        public async Task HandleMessage(SocketMessage messageParam)
+        public async Task HandleMessage(SocketMessage message)
         {
-            var user = ((messageParam as SocketUserMessage)?.Author as SocketGuildUser);
```

Comment:
This should remain. It prevents the bot from executing commands when the message comes from a DM.

You can attach your file upload handler before this if you want to also validate DMs, but that's probably not needed.